### ISSUE TITLE
[BUG] can not return right value in Python 3.10

### ISSUE THREAD

**Issue by hiddify1** [dashingsoft/pyarmor#1008]
This is my test.py file
```python
def test():
    try:
        return 2
    except Exception as e:
        return 1
```
Python 3.10.6
Pyarmor 8.0.12 (trial), 000000, non-profits

Using this command will say `invalid v8 code`
```
pyarmor g test.py
```
by commenting `return 2` it will work

**Comment by jondy** [comment]
got it, thanks.

It will be fixed in next release

**Comment by jondy** [comment]
Fixed in 8.0.15.

Now it's urgent release for Pyarmor 8.0. For next minor version 8.1, it will be full tested before release.

**Comment by joonis** [comment]
This error is still persistent with Pyarmor 8.0.17:

```
Python 3.8.10 (default, Mar 13 2023, 10:26:41) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import mypackage
>>> mypackage.test()
XXX lineno: 2, opcode: 198
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<mypackage.__init__>", line 2, in test
SystemError: unknown opcode
```

**Comment by jondy** [comment]
I can't reproduced it in Python3.8 in `8.1.dev4` with pyarmor.cli.core=2.1.dev1

**Comment by hiddify1** [comment]
I find the error:

if there is no return in the function for all the paths, it will return  None for all the path:

e.g.,
```
def f(x):
   if x=1:
         return 2

f(1)--->None
```

**Comment by jondy** [comment]
Still not reproduce, Both of them return 2 with Python3.8, I'm run in macOS intel.

**Comment by jondy** [comment]
How about in `8.1.dev9`

**Comment by jondy** [comment]
Reproduced and will be fixed in next release.

**Comment by jondy** [comment]
Fixed in `8.1.2`
